motulator/control/grid_following.py

```diff
@@ -95,40 +95,34 @@ def __init__(self, pars):
         self.f_sw = pars.f_sw
         self.L_f = pars.L_f
         self.R_f = pars.R_f
-        self.C_dc = pars.C_dc
-        self.p_max = pars.p_max
-        self.w_0_dc = pars.w_0_dc
-        self.zeta_dc = pars.zeta_dc
-        self.alpha_c = pars.alpha_c
-        self.w_0_pll = pars.w_0_pll
+        # Power references
+        self.p_g_ref = pars.p_g_ref
+        self.q_g_ref = pars.q_g_ref
+        # Activation/deactivation of the DC voltage controller
         self.on_v_dc = pars.on_v_dc
+        # DC voltage reference
         self.u_dc_ref = pars.u_dc_ref
         # Calculated current controller gains:
-        self.k_p_i = pars.alpha_c*pars.L_f-pars.R_f # (-Rf term can be added,
-                                            # see S. Akkari's PhD thesis, 2016)
+        self.k_p_i = pars.alpha_c*pars.L_f-pars.R_f
+        # (-Rf term can be added, see S. Akkari's PhD thesis, 2016)
         self.k_i_i = np.power(pars.alpha_c,2)*pars.L_f
         self.r_i = pars.alpha_c*pars.L_f
         # Definition of the base values
-        self.S_base = pars.S_base
         self.I_base = np.sqrt(2)*pars.S_base/(3*pars.u_gN)
```

Comment:
Consider also removing other base values (I_base, Z_base, L_base, C_base) from controller for simplicity, since it seems that they are not used.